rs/cli/src/ctx.rs

```diff
@@ -39,67 +40,86 @@ pub struct DreContext {
     skip_sync: bool,
     ic_admin_path: Option<String>,
     forum_post_link: Option<String>,
+    dry_run: bool,
```

Comment:
This is a strictly ic-admin thing. Not sure if its needed here?

Reply:
Dry run for other commands, like `dre vote`.

Of course I would prefer if the `--dry-run` flag was (by type) limited to only the commands that support it, but currently it is a global flag.  But that's a bigger refactor.

Reply:
Ok, thanks for clarification